sphinx_immaterial/graphviz.py

```diff
@@ -15,7 +15,7 @@
 import sphinx.ext.graphviz
 import sphinx.util.docutils
 import sphinx.util.logging
-from sphinx.writers.html import HTMLTranslator
+from sphinx.writers.html5 import HTML5Translator
```

Comment:
Seems to me that this change is the only remaining violation in mypy checks. If I revert the change, then the mypy step passes.